src/sales/order-archive.md

```diff
@@ -83,8 +86,20 @@ _[Move Order to Archive]({% link configuration/sales/sales.md %})_
     ![]({% link images/images-ee/order-archived-select-to-restore.png %}){: .zoom}
     _Select Order to be Restored_
 
-1. Set the **Actions** control value to `Remove to Archive`.
+1. Set the **Actions** control value to **Move to Order Management**.
+
+ Look for the message that the archived order has been removed from the archive.
+
+## To export archived order:
+
+1. On the _Admin_ sidebar, click **Sales**.
+
+1. In the _Operations_ section, choose **Orders**.
+
+1. In the button bar, click **Go to Archive**.
+
+1. In the action menu select **Export** drop-down.
 
-    Look for the message that the archived order has been removed from the archive.
+1. Select the desired format.
```

Comment:
Replace with a single step:

1. In the action menu, click **Export**  and select the desired format.